docs/encryption.md

```diff
@@ -17,45 +17,65 @@ Remember that the alt API should only be used over the core API when required by
 
 A very closely related symmetric authenticated encryption scheme (using ChaCha20-Poly1305) has been standardized by the [IETF](https://tools.ietf.org/html/rfc7539).
 
 As a static Diffie-Hellman exchange, the API is slightly different than most asymmetric encryption schemes - for encryption both the recipient public key and sender private key are required, whereas for decryption the recipient private key and sender public key are required. Usually, only the keys of the recipient are required for encryption, though `x25519-xsalsa20poly1305` has the benefit of being an authenticated scheme as well.
 
+The output of the encryption function contains the following:
+* The algorithm in use
+* The public and private keys
+* The original payload
+* The nonce used during encryption
+* The ciphertext
+
+The private & public keys, the nonce and the ciphertext need to be securely stored as they are needed for decryption.
+
 ```js
-// key generation
+// 1. Key generation takes place as no keys are passed as arguments
 
 // callback
 magic.encrypt.pki(message, (err, output) => {
   if (err) { return cb(err); }
   console.log(output);
-  // { alg:        'x25519-xsalsa20poly1305',
-  //   sk:         <Buffer d7 d5 dd 2c 2a eb f1 ... >,
-  //   pk:         <Buffer d2 b2 e2 05 7a 2a ab ... >,
-  //   payload:    <Buffer 41 20 73 63 72 65 61 ... >,
-  //   nonce:      <Buffer b3 4f 59 af 96 e4 4c ... >,
-  //   ciphertext: <Buffer 3c 3d 0e 8b c6 34 83 ... > }
+  /*
+   * {
+   *   alg:        'x25519-xsalsa20poly1305',
+   *   sk:         <Buffer d7 d5 dd 2c 2a eb f1 ... >,
+   *   pk:         <Buffer d2 b2 e2 05 7a 2a ab ... >,
+   *   payload:    <Buffer 41 20 73 63 72 65 61 ... >,
+   *   nonce:      <Buffer b3 4f 59 af 96 e4 4c ... >,
+   *   ciphertext: <Buffer 3c 3d 0e 8b c6 34 83 ... >
+   * }
+   */
 });
 
 // promise
 magic.encrypt.pki(message)
   .then((output) => {
     console.log(output);
-    // { alg:        'x25519-xsalsa20poly1305',
-    //   sk:         <Buffer d7 d5 dd 2c 2a eb f1 ... >,
-    //   pk:         <Buffer d2 b2 e2 05 7a 2a ab ... >,
-    //   payload:    <Buffer 41 20 73 63 72 65 61 ... >,
-    //   nonce:      <Buffer b3 4f 59 af 96 e4 4c ... >,
-    //   ciphertext: <Buffer 3c 3d 0e 8b c6 34 83 ... > }
+  /*
+   * {
+   *   alg:        'x25519-xsalsa20poly1305',
+   *   sk:         <Buffer d7 d5 dd 2c 2a eb f1 ... >,
+   *   pk:         <Buffer d2 b2 e2 05 7a 2a ab ... >,
+   *   payload:    <Buffer 41 20 73 63 72 65 61 ... >,
+   *   nonce:      <Buffer b3 4f 59 af 96 e4 4c ... >,
+   *   ciphertext: <Buffer 3c 3d 0e 8b c6 34 83 ... >
+   * }
+   */
   }).catch((err) => {
     return reject(err);
   });
 });
 
-// supplied key
-const sk = 'd7d5dd...';
-const pk = 'd2b2e2...';
+/*
+ * 2. Encrypt a message using a supplied key pair. Both keys must be 32-byte Buffers
+ * or hex-encoded strings.
+ */
```

Comment:
n.b.: some comments are styled with `/* */`, others with multiple lines of `//`.